tools/isobuild/bundler.js

```diff
@@ -3409,78 +3460,155 @@ async function bundle({
       previousBuilders[arch] = written.builder;
     }
 
-    // Client
-    for (const arch of webArchs) {
-      if (allowDelayedClientBuilds &&
-          hasOwn.call(previousBuilders, arch) &&
-          projectContext.platformList.canDelayBuildingArch(arch)) {
-        // If delayed client builds are allowed, and we have a previous
-        // builder for this arch, and it's an arch that we can safely
-        // build later (e.g. web.browser.legacy), then schedule it to be
-        // built after the server has started up.
-        postStartupCallbacks.push(async ({
-                                           pauseClient,
-                                           refreshClient,
-                                           runLog,
-                                         }) => {
-          const start = +new Date;
-
-          // Build the target first.
-          const target = await makeClientTarget(app, arch, { minifiers });
-
-          // Tell the webapp package to pause responding to requests from
-          // clients that use this arch, because we're about to write a
-          // new version of this bundle to disk. If the message fails
-          // because the child process exited, proceed with writing the
-          // target anyway.
-          await pauseClient(arch).catch(ignoreHarmlessErrors);
-
-          // Now write the target to disk. Note that we are rewriting the
-          // bundle in place, so this work is not atomic by any means,
-          // which is why we needed to pause the client.
-          await writeClientTarget(target);
-
-          // Refresh and unpause the client, now that writing is finished.
-          // If the child process exited for some reason, don't worry if
-          // this message fails.
-          await refreshClient(arch).catch(ignoreHarmlessErrors);
-
-          // Let the webapp package running in the child process know it
-          // should regenerate the client program for this arch.
-          if (Profile.enabled) {
-            runLog.log(`Finished delayed build of ${arch} in ${
-                new Date - start
-            }ms`, { arrow: true });
-          }
-        });
-      } else {
-        // Otherwise make the client target now, and write it below.
-        targets[arch] = await makeClientTarget(app, arch, {minifiers});
+    const clientArchsToProcess = [];
+    
+    if (!isServerOnlyBundle) {
```

Comment:
For better understanding on read this code, isn't it better here to use `isClientOnlyBundle`?